packages/react/runtime/src/lifecycle/patch/commit.ts

```diff
@@ -48,21 +50,36 @@ interface PatchOptions {
 }
 
 function replaceCommitHook(): void {
+  // use our own `options.debounceRendering` to insert a timing flag before render
+  type DebounceRendering = (f: () => void) => void;
+  const injectDebounceRendering = (debounceRendering: DebounceRendering): DebounceRendering => {
+    return (f: () => void) => {
+      debounceRendering(() => {
+        f();
```

Comment:
TODO: Can we mark `vdom_diff_start` before calling this function? To avoid hooking `options[DIFF]`...